client/services/tutors.js

```diff
@@ -1,2 +1,66 @@
-import axios from "axios"
+import api from "./apiConfig"
+
+export const getTutors = async () => {
+  const response = await api.get("/tutors")
+  const tutors = response.data;
+  return tutors;
+};
+
+export const getTutor = async (id) => {
+  const response = await api.get(`/tutors/${id}`)
+  const tutor = response.data;
+  return tutor;
+
```

Comment:
might be a good idea to have try catch statement
if. you not going to use the variable tutor can combine lines 11 and 12, return response.data

Reply:
added try catch
combined 11 and 12